nodejs/index.js

```diff
@@ -25,26 +24,25 @@ app.use(bodyParser.json());
 app.post('/messages', function(req, res) {
   console.log('webhook PAYLOAD:\n', JSON.stringify(req.body, null, 4));
 
-  const appUserId = req.body.appUser._id;
+  const conversationId = req.body.conversation._id;
+  const appId = req.body.app._id;
+
   // Call REST API to send message https://docs.smooch.io/rest/#post-message
   if (req.body.trigger === 'message:appUser') {
-      smooch.appUsers.sendMessage(appUserId, {
-          type: 'text',
-          text: 'Live long and prosper',
-          role: 'appMaker'
-      })
-          .then((response) => {
-              console.log('API RESPONSE:\n', response);
-              res.end();
-          })
-          .catch((err) => {
-              console.log('API ERROR:\n', err);
-              res.end();
-          });
+    sendMessage(appId, conversationId);
+    res.end();
   }
 });
 
 // Listen on port
 app.listen(PORT, () => {
   console.log(`App listening on port ${PORT}`);
 });
+
+async function sendMessage(appId, conversationId){
+    let messagePost = new SunshineConversationsApi.MessagePost();  
+    messagePost.setAuthor({role: 'business'});
+    messagePost.setContent({type: 'text', text: 'Live long and propser'});
+    let response = await apiInstance.postMessage(appId, conversationId, messagePost);
+    console.log('API RESPONSE:\n', response);
+}
```

Comment:
I made it async/await; but most of or code samples are promise-based; I can revert it to promise-based to be consistent.

Reply:
> most of or code samples are promise-based

I think async / await is fine, but I'm curious to which code samples you're referring to

Reply:
The ones that are in the PRs for the v2 Guide docs, for example: https://github.com/smooch/smooch-docs/pull/1104/files#diff-3a65ac4856592db4ae0e0ec67eaf682bL68